consul_awx.py

```diff
@@ -101,7 +102,7 @@ def add_to_group(self, group, host, parent=None):
         # https://docs.ansible.com/ansible/latest/network/getting_started/first_inventory.html
```

Comment:
please move the documentation to the method definition

Reply:
Updated